cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,67 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
+  const userName = 'Bohdan';
+  const userSurname = 'Svystun';
+  const email = 'randomemail@gmail.com';
+  const subject = 'No subjects';
+  it('should successfully register the user with valid credentials', () => {
+    cy.viewport(1920, 1080);
+
+    cy.get('[placeholder="First Name"]').type(userName);
```

Comment:
There are no tests to check that the input fields exist before interacting with them. This can lead to flaky tests. Always assert that elements exist before interacting with them.